src/test/java/org/mule/tools/apikit/AbstractScaffolderTestCase.java

```diff
@@ -76,8 +80,8 @@ protected static String extractName(String path) {
     List<MuleConfig> muleConfigs = new ArrayList<>();
     for (String location : ramlLocations) {
       InputStream muleConfigInputStream = getResourceAsStream(location);
-      MuleConfig muleConfig = fromStream(muleConfigInputStream);
-      muleConfig.setName(location);
+      MuleConfig muleConfig = fromStream(muleConfigInputStream, false);
+      muleConfig.setName(extractName(location).concat(".xml"));
```

Comment:
Is it necessary to extract and add it again? It seems that you need the file name only
```muleConfig.setName(location.substring(location.lastIndexOf("/")));```